packages/now-cli/package.json

```diff
@@ -181,6 +190,6 @@
     "which-promise": "1.0.0",
     "write-json-file": "2.2.0",
     "xdg-app-paths": "5.1.0",
-    "yarn": "1.22.0"
+    "yarn": "1.22.4"
```

Comment:
I thought we don't bundle yarn anymore. Why is this one still here?

Reply:
Removed in a separate PR: https://github.com/zeit/now/pull/4141